composer.lock

```diff
@@ -72,7 +72,7 @@
             },
             "require": {
                 "firebase/php-jwt": "~2.0||~3.0||~4.0||~5.0",
-                "google/apiclient-services": "~0.13",
+                "google/apiclient-services": "dev-master",
```

Comment:
See above, let's revert the `composer.lock` changes.

Reply:
Reverted.